src/Null-Block-Driver.md

```diff
@@ -66,6 +66,68 @@ in this work:
  - [Original RFC Patches](https://github.com/metaspace/linux/tree/null_block-RFC)
  - [Mailing List Post](https://lore.kernel.org/all/20230503090708.2524310-1-nmi@metaspace.dk/)
 
+## 6.10-rc3 Rebase ([`rnull-v6.10-rc3`](https://github.com/metaspace/linux/tree/rnull-v6.10-rc3))
+
+Changes from `rnull-v6.9`:
+
+ - Add `ForeignBorrowed`.
+ - Move `GenDisk` to a builder pattern instead of typestate pattern.
+ - Move block size validation from driver to abstractions.
+ - Pin `NullBlkModuel`.
+ - Refactor `Request::try_set_end`.
+ - Rewrite atomic functions in terms of `core` library functions.
+ - Fix a bug in timer completions where an offset was not calculated correctly.
+ - Refactor `TagSet` initialization in terms of `core::mem::zeroed()` instead of `Opaque::try_ffi_init`
```

Comment:
I am merging it anyway, but in case you want to make it consistent later on:

```suggestion
 - Refactor `TagSet` initialization in terms of `core::mem::zeroed()` instead of `Opaque::try_ffi_init`.
```